django/loader/load/loader.py

```diff
@@ -37,14 +37,23 @@ def load(self, fhir_instance, resource_type):
 
         if isinstance(resource, OperationOutcome):
             resource_id = get_resource_id(fhir_instance)
-            # Increment counter for failed validations
-            counter_failed_validations.labels(resource_id=resource_id, resource_type=resource_type).inc()
+
+            # A bit ugly, DuplicateError should have had code as a class attribute
+            if resource.issue[0].code == DuplicateError("").code:
+                message = "Document already present"
+            else:
```

Comment:
can this be a `else if` to make sure it's a validation error, and add an `else` case to handle all `FHIRStoreError` ?